src/main/java/mods/railcraft/common/core/RailcraftConfig.java

```diff
@@ -335,7 +335,8 @@ private static void loadRecipeOption() {
 
         loadRecipeProperty("minecraft.furnace", "creosote", false, "change to '{t}=true' to add smelting recipes for Creosote Oil to the vanilla furnace");
         loadRecipeProperty("railcraft.track", "useAltRecipes", false, "change to '{t}=true' to use track recipes more similar to vanilla minecraft");
-        loadRecipeProperty("railcraft.bronze", "enableAltBronze", false, "change to '{t}=true' to enable a recipe to craft Bronze Ingots from Tin and Copper Ingots");
+        loadRecipeProperty("railcraft.alloy", "enableAltBronze", false, "change to '{t}=true' to enable a recipe to craft Bronze Ingots from Tin and Copper Ingots, regardless of whether Bronze exists from other mods");
+        loadRecipeProperty("railcraft.alloy", "enableHarderBronze", false, "change to '{t}=true' if you want Bronze recipes to supply 3 Bronze instead of 4");
```

Comment:
Designed to be a config option for hard mode style servers, hence the setnumber (They have their own standard of 3). Later on this would be used for our own alloying system, hence why it's generic, though currently it is only for the alt recipe.